Upstream: Backported commit fd0df6d098 from master by Colin Watson

From 3393c23a8789fb9a2ffd1ec7e19300298bc7ec1f Mon Sep 17 00:00:00 2001
From: Heiko Becker <heirecka@exherbo.org>
Date: Tue, 10 Dec 2013 10:45:38 +0100
Subject: [PATCH] Fix build with freetype>=2.5.1

---
 util/grub-mkfont.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/util/grub-mkfont.c b/util/grub-mkfont.c
index 4e2c5e4..48f39d8 100644
--- a/util/grub-mkfont.c
+++ b/util/grub-mkfont.c
@@ -39,7 +39,7 @@
 #include FT_FREETYPE_H
 #include FT_TRUETYPE_TAGS_H
 #include FT_TRUETYPE_TABLES_H
-#include <freetype/ftsynth.h>
+#include FT_SYNTHESIS_H
 
 #undef __FTERRORS_H__
 #define FT_ERROR_START_LIST   const char *ft_errmsgs[] = { 
-- 
1.8.5.1

